src/js/constants/contentConstants.ts

```diff
@@ -1 +1 @@
-export const aboutUrl = '/public/about.html';
+export const partialAboutUrl = '/public/';
```

Comment:
Replace `'/public/'` with `'/public'`. All of our url constants have no trailling slash.